src/components/LoginForm/LoginForm.jsx

```diff
@@ -132,6 +132,7 @@ export const LoginForm = () => {
             display="block"
             margin="0 auto"
             mt="20px"
+            _hover={isEmpty ? {} : null}
```

Comment:
any disabled button should have no hover behaviour not just the login button

Reply:
done